Apps/Sandcastle/gallery/Side by Side Camera.html

```diff
@@ -0,0 +1,79 @@
+    <meta name="description" content="Use Viewer to start building new applications or easily embed Cesium into existing applications.">
+    <meta name="cesium-sandcastle-labels" content="Beginner, Showcases">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+    #Container {
+          display: flex;
+          width: 100%;
+          height: 100%;
+      }
+      #View3D {
+          display: inline-block;
+          width: 100%;
+      }
+      #View2D {
+          display: inline-block;
+          width: 100%;
+      }
+</style>
+<div id="Container" class="fullSize">
+    <div id="View3D"></div>
+    <div id="View2D"></div>
+</div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+// We create two viewers, a 2D and a 3D one
+// The CSS is set up to place them side by side
+var View3D = new Cesium.Viewer('View3D', { timeline: false });
+var View2D = new Cesium.Viewer('View2D', { timeline: false, sceneMode: Cesium.SceneMode.SCENE2D });
+
+function Sync2DView(){
+    // The center of the view is the point that the 3D camera is focusing on
+    var viewCenter = new Cesium.Cartesian2(Math.floor(View3D.canvas.clientWidth / 2), Math.floor(View3D.canvas.clientHeight / 2));
+    // Given the pixel in the center, get the world position
+    var worldPos = View3D.scene.camera.pickEllipsoid(viewCenter);
+    // Get the distance between the world position of the point the camera is focusing on, and the camera's world position
+    var distance = Cesium.Cartesian3.distance(worldPos, View3D.scene.camera.positionWC);
+    // Tell the 2D camera to look at the point of focus. The distance controls how zoomed in the 2D view is
+    // (try replacing `distance` in the line below with `1e7`. The view will still sync, but will have a constant zoom)
+    View2D.scene.camera.lookAt(worldPos, new Cesium.Cartesian3(0.0, 0.0, distance));
+}
+
+// Apply our sync function every time the 3D camera view changes
+View3D.camera.changed.addEventListener(Sync2DView);
+// By default, the `camera.changed` event will trigger when the camera has changed by 50%
+// To make it more sensitive, we can bring down this sensitivity
+View3D.camera.percentageChanged = 0.01;
+// Setting it to 0 will cause it to trigger every frame and will be equivilent to running this in a `requestAnimationFrame`
```

Comment:
We avoid using (or referencing) `requestAnimationFrame` because Cesium handles the render loop itself. Instead we'd recommend using the [`scene.postUpdate`](https://cesiumjs.org/Cesium/Build/Documentation/Scene.html?classFilter=scene#postRender) event (or another of the render loop related `scene` events).